include/swift/IDE/CodeCompletionResultType.h

```diff
@@ -21,6 +21,15 @@
 namespace swift {
 namespace ide {
 
+enum class AttributeKind : uint8_t {
```

Comment:
`CustomAttributeKind` (or `CustomAttrKind`) sounds more relevant because this is for `CustomAttr`.

Same for `{get|set|}ExpectedAttributeKinds`

Reply:
Changed all occurrences of `AttributeKind` to `CustomAttributeKind`.